src/Native/Unix/CMakeLists.txt

```diff
@@ -4,33 +4,25 @@ project(CoreFX)
 set(CMAKE_MACOSX_RPATH ON)
 set(CMAKE_INSTALL_PREFIX $ENV{__CMakeBinDir})
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=gnu99")
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c11")
```

Comment:
@marek-safar, is this ok for Mono?

Reply:
It's not as Mono requires only `-std=gnu99`

Reply:
This wasn't the required change at all. I just updated it to align with C++11. BTW @marek-safar, is Mono using these cmake files?

Reply:
It doesn't but we want the shared code to use same minimal C version